.github/workflows/docker.yml

```diff
@@ -0,0 +1,115 @@
+name: docker
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # everyday at 10am
+  push:
+    branches: [ master ]
+    tags: [ 'v*.*.*' ]
+  pull_request:
```

Comment:
Why on `pull_request` on master? Merging PR into master is covered by `push`.

Reply:
The `pull_request` allows us to test that build runs successfully (without pushing to DockerHub).